services/storage-users/pkg/config/config.go

```diff
@@ -40,6 +41,11 @@ type Config struct {
 	Tasks             Tasks             `yaml:"tasks"`
 	ServiceAccount    ServiceAccount    `yaml:"service_account"`
 
+	// CLI
+	RevaGatewayGRPCAddr      string `yaml:"gateway_addr" env:"OCIS_GATEWAY_GRPC_ADDR;STORAGE_USERS_GATEWAY_GRPC_ADDR" desc:"The bind address of the gateway GRPC address."`
+	MachineAuthAPIKey        string `yaml:"machine_auth_api_key" env:"OCIS_MACHINE_AUTH_API_KEY" desc:"Machine auth API key used to validate internal requests necessary for the access to resources from other services."`
+	CliMaxAttemptsRenameFile int    `yaml:"max_attempts_rename_file" env:"STORAGE_USERS_CLI_MAX_ATTEMPTS_RENAME_FILE" desc:"Maximum number of attempts to rename a file when user restores the file to existing destination with the same name. The minimum value is 100."`
```

Comment:
```suggestion
	CliMaxAttemptsRenameFile int    `yaml:"max_attempts_rename_file" env:"STORAGE_USERS_CLI_MAX_ATTEMPTS_RENAME_FILE" desc:"The maximum number of attempts to rename a file when a user restores a file to an existing destination with the same name. The minimum value is 100."`
```
Questions:
* A minimum number of attempts - 100.
What happens when exceeding?
* Where does 100 come from? I do not see that as default defined.
or maybe I just missed that...